File: sdkwork-backend-sdk-go/types/generate_voice_speaker_vo.go
package types

// 语音说话人生成结果VO
type GenerateVoiceSpeakerVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	RequestId string `json:"requestId"`
	TaskId string `json:"taskId"`
	TaskStatus string `json:"taskStatus"`
	VoiceResources AudioMediaResourceList `json:"voiceResources"`
	VoiceSpeakerId string `json:"voiceSpeakerId"`
	VoiceSpeakerName string `json:"voiceSpeakerName"`
	VoiceType string `json:"voiceType"`
	VoiceStyle string `json:"voiceStyle"`
	Model string `json:"model"`
	Duration int `json:"duration"`
	ErrorMessage string `json:"errorMessage"`
}

File: sdkwork-backend-sdk-csharp/Models/GenerateVoiceSpeakerVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Backend.Models
{
    public class GenerateVoiceSpeakerVO
    {
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
        public string? RequestId { get; set; }
        public string? TaskId { get; set; }
        public string? TaskStatus { get; set; }
        public AudioMediaResourceList? VoiceResources { get; set; }
        public string? VoiceSpeakerId { get; set; }
        public string? VoiceSpeakerName { get; set; }
        public string? VoiceType { get; set; }
        public string? VoiceStyle { get; set; }
        public string? Model { get; set; }
        public int? Duration { get; set; }
        public string? ErrorMessage { get; set; }
    }
}
